client/milvusclient/read.go

```diff
@@ -228,6 +228,38 @@ func (c *Client) HybridSearch(ctx context.Context, option HybridSearchOption, ca
 	return resultSets, err
 }
 
+func (c *Client) RunAnalyzer(ctx context.Context, option RunAnalyzerOption, callOptions ...grpc.CallOption) ([][]*Token, error) {
```

Comment:
returning `[][]*Token` could be to restricted. If the run analyzer want to return more info for each text import, we have to change method signature or add new method here. maybe add a wrapper for []*Token (the result of one text input)